packages/editor-ui/src/components/CodeNodeEditor/CodeNodeEditor.vue

```diff
@@ -156,34 +151,52 @@ export default mixins(linterExtension, completerExtension, workflowHelpers).exte
 				if (!viewUpdate.docChanged) return;
 				this.trackCompletion(viewUpdate);
 				this.$emit('valueChanged', this.content);
+				if (this.content === this.defaultValue) {
+					this.isDefault = true;
+				} else {
+					this.isDefault = false;
+				}
```

Comment:
```suggestion
				this.isDefault = this.content === this.defaultValue;
```

Reply:
haha, yes that happens if you refactor and simplify code and then do not think if it can be simplified even more ;-)

Reply:
![image](https://user-images.githubusercontent.com/196144/223978296-44b25290-2802-47d6-b010-0d8bf2b2acd4.png)